### DISCUSSION TITLE
Welcome to ucx Discussions!

### DISCUSSION THREAD

**Discussion by nfx** [databrickslabs/ucx#1180] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ganeshgirase** [comment]
Hello @nfx,

Thanks for the taking this initiative. 

Do we have any documentation or manual exists so contributor can use that to run these workflows to test at local environment?

Thanks,
Ganesh

**Comment by sapdbajaj** [comment]
Could I use this tool to 
- migrate foreign glue catalog to managed UC catalog
We know internally glue uses hive as metastore, hence wonder if this tool would allow to migrate from foreign glue catalog (on UC) to managed UC catalog.
